worker/src/telegram/resources/sql/enqueue-messages-telegram.sql

```diff
@@ -0,0 +1,70 @@
+CREATE OR REPLACE FUNCTION enqueue_messages_telegram(job_id int)
+RETURNS VOID LANGUAGE plpgsql AS $$
+DECLARE
+  selected_campaign_id int;
+BEGIN
+  UPDATE job_queue
+  SET status = 'SENDING', updated_at = clock_timestamp()
+  WHERE id = job_id
+    AND status = 'ENQUEUED'
+	RETURNING campaign_id INTO selected_campaign_id;
+
+  WITH messages AS (
+    -- start with this campaign's messages joined with telegram ids
```

Comment:
Why not start with getting the list of subscribers for this campaign's botid? This is presumably a much shorter list than the entirety of telegram_subscribers table. 

Join the messages with this bot's subscribers. 

Then set the error "Bot subscription not found" for the messages without a corresponding subscriber, without regard to the existence of a telegram id. 

Enqueue the messages with corresponding subscribers.

To the admin, does knowing whether someone has a telegram id matter? If they want to subscribe to the bot, they have to get a telegram id anyway.